src/DevLicenseDimo.sol

```diff
@@ -205,10 +216,9 @@ contract DevLicenseDimo is DevLicenseMeta {
 
         address clientId = _tokenIdToClientId[tokenId];
         delete _tokenIdToClientId[tokenId];
+        delete _tokenIdToAlias[tokenId];
```

Comment:
What about the reverse association? Do we need to delete that too?

Reply:
Shame on me